src/types.rs

```diff
@@ -499,7 +501,19 @@ pub struct CompleteObjectInfo {
 }
 
 impl From<RawObjectInfo> for CompleteObjectInfo {
-    fn from(raw: RawObjectInfo) -> Self {
+    fn from(mut raw: RawObjectInfo) -> Self {
+        if let Some(ref raw_id) = raw.debug_id {
+            if let Ok(id) = DebugId::from_breakpad(raw_id.as_str()) {
+                raw.debug_id = Some(id.to_string());
+            }
+        }
+
+        if let Some(ref raw_id) = raw.code_id {
+            if let Ok(id) = raw_id.parse() {
```

Comment:
This parses `raw_id` into a String, which is a no-op. Run this instead:

```suggestion
            if let Ok(id) = raw_id.parse::<CodeId>() {
```

Reply:
Good catch...